### PR TITLE
[MXNET-1212] Documentation fix on the SSD example

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##
Documentation change regarding the conversation in this PR: https://github.com/apache/incubator-mxnet/pull/13162
@nswamy @andrewfayres @zachgk 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer/objectdetector/README.md [hunk 1]
```diff
@@ -11,15 +11,13 @@ The model is trained on the [Pascal VOC 2012 dataset](http://host.robots.ox.ac.u
...
-6. [Next steps](#next-steps)
 
 
 ## Prerequisites
 
 1. MXNet
-2. MXNet Scala Package
-3. [IntelliJ IDE (or alternative IDE) project setup](http://mxnet.incubator.apache.org/tutorials/scala/mxnet_scala_on_intellij.html) with the MXNet Scala Package
+2. MXNet Scala/Java Package
+3. [IntelliJ IDE (or alternative IDE) project setup](https://github.com/apache/incubator-mxnet/blob/master/docs/tutorials/java/mxnet_java_on_intellij.md) with the MXNet Scala Package
```
Comment (piyushghai) [inline scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer/objectdetector/README.md]: with the MXNet Java Package.

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by piyushghai** [comment]
The same files are also addressed in #13277  ?

**Comment by lanking520** [comment]
Since @andrewfayres address this as well, this one will be closed

**Pull request closed without merging**